Admin/CrossTenantMailboxMigrationValidation.ps1

```diff
@@ -430,6 +467,22 @@ function CheckObjectsSourceOffline {
             } else {
                 Write-Verbose -Message "No aux archives are present on SOURCE mailbox"
 
+                #Check for the T2T license on any of the objects (either source or target) as long as the source mailbox is a regular mailbox
```

Comment:
This looks like what we have in Line 175. Would be good to create a separate function which is then called to avoid duplicate code.